babel.config.js

```diff
@@ -1,5 +1,7 @@
 const packageJson = require('./packages/cozy-client/package.json')
 
+console.log('oto')
```

Comment:
I knew it! https://en.wikipedia.org/wiki/Ordo_Templi_Orientis

Reply:
You got me !